packaging/selinux/microshift.fc

```diff
@@ -1,6 +1,8 @@
 # vim: sw=8:ts=8:et
-/var/run/secrets/kubernetes.io/serviceaccount(/.*)?				gen_context(system_u:object_r:container_runtime_tmpfs_t,s0)
-/var/lib/microshift/certs/ca-bundle(/.*)?                                       gen_context(system_u:object_r:container_file_t,s0)
-/usr/local/bin/microshift                                                  --	gen_context(system_u:object_r:container_runtime_exec_t,s0)
-/usr/bin/microshift                                                        --	gen_context(system_u:object_r:container_runtime_exec_t,s0)
-/var/hpvolumes(/.*)?            						gen_context(system_u:object_r:container_file_t,s0)
+/var/run/secrets/kubernetes.io/serviceaccount(/.*)?             gen_context(system_u:object_r:container_runtime_tmpfs_t,s0)
+/var/lib/microshift/certs/ca-bundle(/.*)?                       gen_context(system_u:object_r:container_file_t,s0)
+/var/lib/microshift-backups(/.*)?                           	gen_context(system_u:object_r:container_var_lib_t,s0)
+/etc/microshift(/.*)?                                       	gen_context(system_u:object_r:kubernetes_file_t,s0)
+/usr/local/bin/microshift                                   --	gen_context(system_u:object_r:container_runtime_exec_t,s0)
+/usr/bin/microshift                                         --	gen_context(system_u:object_r:container_runtime_exec_t,s0)
```

Comment:
Have we determined why `container_runtime_exec_t` appropriate for microshift, when e.g. kubelet is unconfined on OCP?

Reply:
This has been updated, we are going to be a little more restrictive that OCP and run as `kubelet_exec_t`, That seems to make more sense for our use case.